.eslintrc

```diff
@@ -1,3 +1,4 @@
 {
-  "extends": "next"
+  "extends": ["next", "plugin:jsx-a11y/recommended"],
```

Comment:
I upgraded to Next.js 11  and ESLint works there out of the box https://nextjs.org/docs/basic-features/eslint

Reply:
are there any plugins you'd recommend?